types/args/index.d.ts

```diff
@@ -0,0 +1,77 @@
+// Type definitions for args 3.0
+// Project: https://github.com/leo/args#readme
+// Definitions by: Slessi <https://github.com/Slessi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const c: args;
+export = c;
+
+interface args {
+    sub: string[];
+
+    option(name: string | [string, string], description: string, defaultValue?: any, init?: OptionInitFunction): args;
+    options(list: Option[]): args;
+    command(name: string, description: string, init?: CommandInitFunction, aliases?: string[]): args;
```

Comment:
can you inline these the types for MriUnknownFunction, MinimistUnknownFunction and other such rather than aliasing them since they seem to be used just once

Reply:
I moved them as inline except for `OptionInitFunction` which is used twice